civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -7195,3 +7195,16 @@ function civihr_employee_portal_delete_report_configuration_json($reportName, $c
   print drupal_json_output(array('status' => 'error'));
   drupal_exit();
 }
+
+/**
+ * Changes the favicon for both the SSP and CiviHR admin
```

Comment:
Change to :
/**
  Implements hook_html_head_alter().
  Changes the favicon for both the SSP and CiviHR admin.